rpcs3/Emu/RSX/gcm_enums.cpp

```diff
@@ -143,9 +143,8 @@ void fmt_class_string<CellGcmLocation>::format(std::string& out, u64 arg)
 		case CELL_GCM_CONTEXT_DMA_MEMORY_HOST_BUFFER: return "Main-Buffer"; // Main memory for DMA operations
 		case CELL_GCM_CONTEXT_DMA_REPORT_LOCATION_LOCAL: return "Report Local";
 		case CELL_GCM_CONTEXT_DMA_REPORT_LOCATION_MAIN: return  "Report Main";
-		case CELL_GCM_CONTEXT_DMA_NOTIFY_MAIN_0: return "Notify"; // TODO
-
-		//case CELL_GCM_CONTEXT_DMA_TO_MEMORY_GET_NOTIFY0:
+		case CELL_GCM_CONTEXT_DMA_NOTIFY_MAIN_0: return "_Notify";
+		case CELL_GCM_CONTEXT_DMA_TO_MEMORY_GET_NOTIFY0: return "_Get-Notify";
```

Comment:
Its a todo because there are 8 locations for each, not just one. _Notify0, _Notify1, _Notify2 etc.

Reply:
Fixed. What about CELL_GCM_CONTEXT_DMA_TO_MEMORY_GET_NOTIFY0? Is it only one?

Reply:
Same thing, 8 locations.

Reply:
What are their codes then?

Reply:
Decrementing 7 times from origin value, one for each.

Reply:
This is a bit strange. 0x66604200 for 0 becomes 0x666041FF, etc.

Reply:
Original value must be 0x66604200 then.

Reply:
Hmm, ok... it doesn't seem used anyway.